RCAIDE/Library/Methods/Propulsors/Converters/Combustor/compute_combustor_performance.py

```diff
@@ -0,0 +1,75 @@
+# RCAIDE/Library/Methods/Propulsors/Converters/Combustor/compute_combustor_performance.py
+# (c) Copyright 2023 Aerospace Research Community LLC
+# 
+# Created:  Jun 2024, M. Clarke  
+
+# ---------------------------------------------------------------------------------------------------------------------- 
+# compute_combustor_performance
+# ----------------------------------------------------------------------------------------------------------------------    
+def compute_combustor_performance(combustor,conditions):
```

Comment:
Group conversation results: The conditions track the components conditions

compute_combustor_performance(combustor,conditions)

return

Reply:
done